app/src/main/java/org/p2p/wallet/sell/ui/payload/SellPayloadPresenter.kt

```diff
@@ -12,48 +24,48 @@ import org.p2p.wallet.user.interactor.UserInteractor
 import org.p2p.wallet.utils.toBase58Instance
 import timber.log.Timber
 import java.math.BigDecimal
-import kotlinx.coroutines.launch
+import java.math.RoundingMode
 
-private val MIN_AMOUNT_TO_SELL = BigDecimal.valueOf(20)
-private val MAX_AMOUNT_TO_SELL = BigDecimal.valueOf(100)
+private const val SELL_QUOTE_REQUEST_DEBOUNCE_TIME = 10_000L
 
 class SellPayloadPresenter(
     private val sellInteractor: SellInteractor,
-    private val userInteractor: UserInteractor,
     private val tokenKeyProvider: TokenKeyProvider,
-    private val moonpayWidgetUrlBuilder: MoonpayWidgetUrlBuilder
+    private val moonpayWidgetUrlBuilder: MoonpayWidgetUrlBuilder,
+    private val userInteractor: UserInteractor,
+    private val resourceProvider: ResourcesProvider
 ) : BasePresenter<SellPayloadContract.View>(),
     SellPayloadContract.Presenter {
 
-    private sealed interface ViewState {
-        object Initial : ViewState
```

Comment:
how do you handle returning back from the Moonpay without finishing flow? The inputs still there?

Reply:
hmm I don't understand what you mean